web/styles/media.css

```diff
@@ -1,3 +1,4 @@
 @custom-media --small-up (min-width: 390px);
 @custom-media --medium-up (min-width: 768px);
 @custom-media --large-up (min-width: 1025px);
+@custom-media --session-large-up (min-width: 1310px);
```

Comment:
Why is there a custom breakpoint defined specifically for the Sessions page? Can't the Figma sketches be adjusted to be consistent with the conventional breakpoints?

Reply:
This is explained in the PR description ("Description" section, 2nd paragraph).

Originally, I intentionally set the breakpoints so that there would only ever be _more_ space available than in Figma. But this meant the "tablet" design applied to a lot of configurations, so it was adjusted based on Sindre's input. (Also, there's no sketch for <390px anyway.)

I don't think it's so easy to adjust the desktop sketch to work down to 1025px without basically throwing out/redoing most of it, and I suppose it's already been approved, so that's why we ended up with this. Maybe sometime after Easter we should discuss a somewhat less waterfall-y approach to things (also it would be nice if the desktop sketch was made at 1025px width, and similarly for the other variants, but this is presumably a big job to re-do now…)